front/src/modules/stdcmAllowances/components/StdcmAllowances.tsx

```diff
@@ -6,24 +6,19 @@ import { useTranslation } from 'react-i18next';
 import InputSNCF from 'common/BootstrapSNCF/InputSNCF';
 import InputGroupSNCF, { InputGroupSNCFValue } from 'common/BootstrapSNCF/InputGroupSNCF';
 
-import {
-  getGridMarginBefore,
-  getGridMarginAfter,
-  getStandardStdcmAllowance,
-} from 'reducers/osrdconf/selectors';
-import {
-  updateGridMarginAfter,
-  updateGridMarginBefore,
-  updateStdcmStandardAllowance,
-} from 'reducers/osrdconf';
 import { StandardAllowance } from 'applications/operationalStudies/consts';
 import { AllowanceValue } from 'common/api/osrdEditoastApi';
 import { convertInputStringToNumber } from 'utils/strings';
 import { ALLOWANCE_UNITS_KEYS } from 'applications/stdcm/components/allowancesConsts';
+import { useOsrdActions, useOsrdSelectors } from 'common/osrdContext';
+import { stdcmConfSliceActionsType } from 'reducers/osrdconf/stdcmConf';
+import { stdcmConfSelectorsType } from 'reducers/osrdconf/stdcmConf/selectors';
```

Comment:
Could you also update the other type imports of the file please
```suggestion
import type { stdcmConfSliceActionsType } from 'reducers/osrdconf/stdcmConf';
import type { stdcmConfSelectorsType } from 'reducers/osrdconf/stdcmConf/selectors';
```

Reply:
Done